uisdk/src/main/res/drawable/kh_uisdk_ic_calendar.xml

```diff
@@ -0,0 +1,34 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Can we update the `uisdk_ic_prebook` (rename it if you want to add the `kh_` prefix instead of new file?

It might be better to introduce a color e.g. @color/kh_uisdk_prebook_icon" or @color/kh_uisdk_icon_stroke" instead of referencing @color/kh_uisdk_colorAccent" inside this asset. This would enable demand partners to customise the color without changing the accent color